docs/docsite/rst/administration/index.rst

```diff
@@ -1,10 +1,8 @@
 .. _ag_start:
 
-==================
-AWX Administration
-==================
-
-AWX Administration
+=========================
+Administration Guide TOC
```

Comment:
@oraNod - Well, I thought it was redundant when you have the "caption" and the doc header read the same, and on the side nav it makes better sense to put TOC because that's what the index file is. But if repeating the name of the doc maintains consistency with the other upstream docs, then I'm ok with whatever you suggest.